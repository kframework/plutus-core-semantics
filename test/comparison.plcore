(program

(module Foo (imported) (exported () ()) (local () ())
  (define c1 (builtin lessThanInteger 1 2))
  (define c2 (builtin lessThanInteger 2 2))
)

)
