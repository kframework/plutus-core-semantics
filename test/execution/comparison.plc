(program

(module Foo (import) (export () ())
  (define c1 (builtin lessThanInteger 1 2))
  (define c2 (builtin lessThanInteger 2 2))
)

)
