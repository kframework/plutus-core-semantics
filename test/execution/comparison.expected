<T>
  <k>
    .
  </k>
  <lenv>
    .Map
  </lenv>
  <genv>
    Foo . c1 |-> ( con Prelude . True .ValList )
    Foo . c2 |-> ( con Prelude . False .ValList )
  </genv>
  <modules>
    ModuleCellSetItem ( <Module>
      <curr>
        0
      </curr>
      <name>
        Foo
      </name>
      <dataCons>
        .TyExpList
      </dataCons>
      <decl>
        .QualCList
      </decl>
      <importedModules>
        .ModNmList
      </importedModules>
      <exportTy>
        .TyExpList
      </exportTy>
      <exportTm>
        .QualCList
      </exportTm>
    </Module> )
  </modules>
</T>
